src/main/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/infrastructure/client/HomeOfficeSendNotificationClient.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientResponseException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCase;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities.AsylumCaseDefinition;
+import uk.gov.hmcts.reform.iahomeofficeintegrationapi.infrastructure.client.util.HomeOfficeApiUtil;
+
+@Service
+public class HomeOfficeSendNotificationClient {
+
+    private final RestTemplate restTemplate;
+    private final String homeOfficeSendInstructUrl;
+    private HomeOfficeApiUtil homeOfficeApiUtil;
+
+    public HomeOfficeSendNotificationClient(
+        @Value("${homeoffice.endpoint}") String homeOfficeEndpoint,
+        @Value("${homeoffice.instruct.uri}") String homeOfficeSendInstructUrl,
+        RestTemplate restTemplate,
+        HomeOfficeApiUtil homeOfficeApiUtil
+    ) {
+        this.restTemplate = restTemplate;
+        this.homeOfficeSendInstructUrl = homeOfficeEndpoint + homeOfficeSendInstructUrl;
+        this.homeOfficeApiUtil = homeOfficeApiUtil;
+    }
+
+    public int sendNotification(AsylumCase asylumCase) {
+
+        try {
+            ResponseEntity<String> response = restTemplate.postForEntity(
```

Comment:
this response is String, do we have more specific response like JSON model which we can interpret?

Reply:
Has this one been addressed somewhere?  Maybe in a different PR in which case please comment in here.

Reply:
This is changed in PR #77